src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -197,6 +218,39 @@ private static String getMypyConfigFile(Project project, String mypyConfigFilePa
         return mypyConfigFilePath;
     }
 
+    public static String detectVenvMypyPath(Project project) throws IOException {
+        VirtualFile interpreterFile = getInterpreterFile(project);
+        String interpreterPath = interpreterFile != null ? interpreterFile.getPath() : null;
+        if (interpreterPath == null) {
+            return null;
+        }
+        String venvPath =
+                Paths.get(PathUtil.getParentPath(PathUtil.getParentPath(interpreterPath)), "Scripts").toString();
+        return findFileByName("mypy", venvPath, "glob:**/*.exe");
```

Comment:
I think we should not have the mypy executable name defined in multiple places.
There are already 2 constants that define this executable:
```Java
public static final String MYPY_PACKAGE_NAME = "mypy";
private static final String MYPY_EXECUTABLE_NAME = MYPY_PACKAGE_NAME + (OS.isWindows() ? ".exe" : "");
```
Could you please create a new constant for the extension and use the `MYPY_PACKAGE_NAME` constant instead of hardcoding it as a string?

Something like this:
```Java
public static final String MYPY_PACKAGE_NAME = "mypy";
private static final String MYPY_EXECUTABLE_EXTENSION = OS.isWindows() ? ".exe" : "";
private static final String MYPY_EXECUTABLE_NAME = MYPY_PACKAGE_NAME + MYPY_EXECUTABLE_EXTENSION;
```
```Java
String venvPath =
   Paths.get(PathUtil.getParentPath(PathUtil.getParentPath(interpreterPath)), "Scripts").toString();
return findFileByName(MYPY_PACKAGE_NAME, venvPath, "glob:**/*" + MYPY_EXECUTABLE_EXTENSION);
```